docs/sources/migration-guide/migrating-from-thanos-or-prometheus.md

```diff
@@ -47,53 +47,33 @@ aws s3 cp <DIRECTORY> s3://<TENANT>/<DIRECTORY>
 gsutil -m cp -r <DIRECTORY> gs://<TENANT>/<DIRECTORY>
 ```
 
-## Migrating the block `meta.json` metadata using `metaconvert`
+## Block metadata
 
 Every block has a `meta.json` metadata file used by Grafana Mimir, Prometheus, and Thanos to identify the block contents.
 Each project has its own metadata conventions.
-The `metaconvert` tool migrates the `meta.json` metadata from one project to another.
 
-### Downloading `metaconvert`
+Ingesters from Mimir 2.1 and earlier added an external label into `meta.json` file to identify the tenant that owns the block.
+Mimir compactor also adds special label identifying compactor-shard when split-and-merge sharding feature is enabled.
 
-- If you are using Docker, run the following command:
+In Mimir 2.2 and later blocks no longer have a label identifying the tenant. Compactor shard label is still used.
 
-```bash
-docker pull grafana/metaconvert:latest
-```
-
-- If you are using a release binary, download the appropriate release asset for your operating system and architecture and make it executable. For Linux with the AMD64 architecture, run the following command:
-
-```bash
-curl -LO https://github.com/grafana/mimir/releases/latest/download/metaconvert-linux-amd64
-chmod +x metaconvert-linux-amd64
-```
-
-- If you are using Go, run the following command:
-
-```bash
-go install github.com/grafana/mimir/cmd/metaconvert@latest
-```
-
-### Running `metaconvert`
-
-> **Warning:** The `metaconvert` tool modifies objects in place.
-> Ensure you enable bucket versioning or have backups before running the tool.
-
-To run `metaconvert`, provide it with the bucket configuration. Use `metaconvert -h` to list available parameters.
+When querying blocks, Mimir ignores all external labels and doesn't add them to the results.
```

Comment:
```suggestion
When querying blocks, Mimir ignores all external labels and doesn't add them to the results.
```
What does "When querying blocks" mean?
* Does it get blocked (by what?)
* Does it block something? In either case, who or what, and by whom?

Reply:
> What does "When querying blocks" mean?
> 
> * Does it get blocked (by what?)
> * Does it block something? In either case, who or what, and by whom?

"blocks" is used as a noun here, not verb. When user sends a query to execute, we say that "Mimir queries (TSDB) blocks".